pkg/reconcilermanager/controllers/reconciler_base.go

```diff
@@ -424,11 +424,11 @@ func (r *reconcilerBase) deployment(ctx context.Context, dRef client.ObjectKey)
 	return deployObj, nil
 }
 
-func mountConfigMapValuesFiles(templateSpec *corev1.PodSpec, c *corev1.Container, valuesFrom []v1beta1.ValuesFileSources) {
+func mountConfigMapValuesFiles(templateSpec *corev1.PodSpec, c *corev1.Container, valuesFileRefs []v1beta1.ValuesFileRefs) {
```

Comment:
please add a func comment explaining what this does

Reply:
done